Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -94,13 +107,31 @@ protected override bool IsValidController(IMixedRealityController controller)
                             }
                         }
                     }
+
+                    // Check if the palm angle meets the prescribed threshold
+                    palmFacingThresholdMet = palmCameraAngle < facingCameraTrackingThreshold;
+
+                    // If using hybrid hand rotation, we proceed with additional checks
+                    if (followHandUntilFacingCamera && palmFacingThresholdMet)
```

Comment:
nit: Reminder once you create the public properties, use those here and not the private instances

Reply:
Done.